master/buildbot/test/unit/test_reporters_irc.py

```diff
@@ -53,6 +53,14 @@ def makeBot(self, *args, **kwargs):
             args = ('nick', 'pass', ['#ch'], [], [], {})
         return irc.IrcStatusBot(*args, **kwargs)
 
+    def test_groupDescribe(self):
+        b = self.makeBot()
```

Comment:
`b` is not really descriptive and, while I understand it stands for `bot`, I do not see why it could not be named so...

Reply:
I think it is unfair to penalize @zware, when he is just trying to keep the existing style of the file.